components/dashboard/src/admin/Settings.tsx

```diff
@@ -22,11 +22,14 @@ export default function Settings() {
     const { user } = useContext(UserContext);
 
     useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
```

Comment:
Do we still need to disable it since the missing dependency arg is what's causing the infinite loop?

Reply:
I think yes, because we should not be giving our  users insights or the permission to see/set this page for the whole SAAS instance? 🤔

Reply:
It wasn't causing an infinite loop. 😌 It's just on each re-render it was called, instead of a single init call.

Reply:
Not sure, if it's only the missing dependency arg. It could also be that this query is too expensive with our large instance DB:

https://github.com/gitpod-io/gitpod/blob/5f6114c2823997874352f8cd8f2207fef1253f29/components/gitpod-db/src/typeorm/workspace-db-impl.ts#L337-L344

Don't know how to figure that out, though.

Reply:
You are right. The useEffect issue was not a big deal, it just amplified the actual problem.